packages/aws-cdk/lib/api/logs/logs-monitor.ts

```diff
@@ -124,10 +124,10 @@ export class CloudWatchLogEventMonitor {
   }
 
   private async tick(): Promise<void> {
+    // excluding from codecoverage because this
```

Comment:
This is bizarre but i'm seeing sporadic behavior with respect to coverage of the `if` statement itself. 

Our current [report](https://app.codecov.io/github/aws/aws-cdk/blob/main/packages%2Faws-cdk%2Flib%2Fapi%2Flogs%2Flogs-monitor.ts#L127) says its uncovered - but how can that be if the other lines of function ARE covered? 
Lets disable this until we get to the bottom of it because it introduces sporadic coverage. 

Followup to https://github.com/aws/aws-cdk/pull/32107.